File: src/test/java/br/com/model/entities/classes/NovoTelefoneTest.java
package br.com.model.entities.classes;

import org.junit.jupiter.api.Test;

import br.com.newObjects.model.entities.classes.NovoTelefone;
import br.com.newObjects.model.entities.classes.usuario.NovoFornecedor;

public class NovoTelefoneTest {

    @Test
    void testNovaListaTelefone() {
        System.out.println(NovoTelefone.novaListaTelefones(NovoFornecedor.novoFornecedor()));
    }
}

File: src/test/java/br/com/model/entities/classes/NovoTelefoneTest.kt
package br.com.model.entities.classes

import br.com.newObjects.model.entities.classes.usuario.NovoFornecedor
import org.junit.jupiter.api.Test

class NovoTelefoneTest {
    @Test
    fun testNovaListaTelefone() {
        val fornecedor = NovoFornecedor.novoFornecedor()
        println(fornecedor!!.telefones)
    }
}
